sdk/tables/Azure.Data.Tables/tests/TableServiceLiveTestsBase.cs

```diff
@@ -262,7 +262,7 @@ protected async Task<TResult> CosmosThrottleWrapper<TResult>(Func<Task<TResult>>
                     {
                         throw;
                     }
-                    await Task.Delay(750);
+                    await Task.Delay(5000);
```

Comment:
This looks fishy.  :smile:   5 seconds is kind of a long time.  Might be worth starting `int delay = 500;` outside the loop and then updating `delay *= 2;` after each delay.  And probably changing the maximum retryCount to something like `5` so we have .5s, 1s, 2s, 4s, and 8s delays before giving up.  This will stay snappy when things are good but let us slow down when it's really needed.